compose/network.go

```diff
@@ -79,9 +96,6 @@ func (n *Networks) New(project string) (string, error) {
 	for i := 0; i < 2; i++ {
 		now = time.Now()
 		subnet, err := _network.NextAvailableNetwork(subnets, n.min, n.max, n.mask)
```

Comment:
what happens if `err != nil` ? subnet is unusable.

Reply:
Err nil check added